src/library/VTKExtensions/Applicative/Testing/TestF3DObjectFactory.cxx

```diff
@@ -0,0 +1,10 @@
+#include <vtkNew.h>
+#include <vtkTestUtilities.h>
+
+#include "vtkF3DObjectFactory.h"
+
+int TestF3DObjectFactory(int argc, char* argv[])
+{
+  vtkNew<vtkF3DObjectFactory> factory;
```

Comment:
maybe check to instantiate a `vtkPolyDataMapper` and check that we have a `vtkF3DPolyDataMapper`